### DISCUSSION TITLE
如何在自己的数据集上进行微调

### DISCUSSION THREAD

**Discussion by Sherryran08** [RapidAI/TableStructureRec#90] [Q&A]
您好，想问一下如何根据提供的模型参数在自己的数据集上进行微调以及数据集的格式需要参考什么格式？感谢您的回答！

**Comment by Joker1212** [comment]
有线表格的话，模型是别人贡献的，想微调可以参考unet神经网络项目https://github.com/jiangnanboy/table_structure_recognition，无线表格直接是读光的模型
